sidecar/src/service/session_info.rs

```diff
@@ -143,6 +144,17 @@ impl SessionInfo {
     {
         f(&mut self.get_trace_config());
     }
+
+    pub(crate) fn get_dogstatsd(&self) -> MutexGuard<dogstatsd::Flusher> {
+        self.dogstatsd.lock().unwrap()
```

Comment:
I think we should use `expect()` instead of `unwrap()`, however unlikely it is that we'll panic (this applies to the entire PR, not just this one line).

Reply:
If you don't mind, I'd prefer to do it in a separate PR.
Because for example, `get_dogstatsd` is similar to many other methods of the `SessionInfo` struct, which all use `unwrap`

Reply:
@iamluc - A separate PR is ok.